core/src/main/java/org/infinispan/container/entries/VersionedRepeatableReadEntry.java

```diff
@@ -21,12 +21,12 @@
  * @author Manik Surtani
  * @since 5.1
  */
-public class ClusteredRepeatableReadEntry extends RepeatableReadEntry implements Versioned {
+public class VersionedRepeatableReadEntry extends RepeatableReadEntry implements Versioned {
```

Comment:
:+1:. It think this class was from the time where the version wasn't in the Metadata.
Are you going to remove it in this PR or another one?